### DISCUSSION TITLE
Remove key:value that is in uknown level of nesting

### DISCUSSION THREAD

**Discussion by jwalkerbg** [nlohmann/json#3496] [Q&A] [answered]
Hello,
My case is: I have a json schema that have objects at several levels of nesting. This schema may have pairs with key "example", and it is not previously known if they exist and where in the hierarchy exist and how much they are. Is it possible to have an hierarchical iterator that find these pairs? In my case, I have to remove these, nevertheless the values are simple types or sub-objects or arrays.

Thank you
Ivan Cenov

**Comment by nlohmann** [comment, marked as answer]
You can write code similar to https://stackoverflow.com/a/46630394/266378.

**Reply by nlohmann** [reply]
@jwalkerbg Do you need further assistance?
